acceptance/new-user/join-button.test.js

```diff
@@ -0,0 +1,64 @@
+/** @jest-environment jsdom */
+const puppeteer = require("puppeteer");
+const URLS = require("../../constants/urls");
+const { JOIN } = URLS;
+const { delay } = require("../../constants/utils");
+
+let browser, page;
+
+async function confirmAlerts() {
```

Comment:
since this is called only once and we are not reusing it. I don't see any point in creating it as a separate function

Reply:
It is keeping the code clean and there are some navigation issues while changing the test in one.